documentation/modules/exploit/linux/http/opennetadmin_ping_cmd_injection.md

```diff
@@ -0,0 +1,66 @@
+OpenNetAdmin provides a database managed inventory of your IP network. Each subnet, host, and IP can be tracked via a centralized AJAX enabled web interface that can help reduce tracking errors.
+
+This module exploits a command injection in OpenNetAdmin.
+
+## Verification
```

Comment:
Please move the `Verification` section after the `Description` and `Vulnerable Application` sections, for consistency with existing documentation.